python/mleap/pyspark/feature.py

```diff
@@ -0,0 +1,38 @@
+from pyspark.ml.util import JavaMLReadable, JavaMLWritable, _jvm
+from pyspark.ml.wrapper import JavaTransformer
+from pyspark.ml.param.shared import HasInputCol, HasOutputCol
+
+
+class StringMap(JavaTransformer, HasInputCol, HasOutputCol, JavaMLReadable, JavaMLWritable):
```

Comment:
maybe this should be in a string_map.py file, as we're likely to add support for some more transformers?